.azure/production-pipelines.yml

```diff
@@ -22,13 +22,4 @@ stages:
   jobs:
   - job: build
     steps:
-    - template: templates/build.yml
-
-- stage: test
-  dependsOn: build
-  displayName: Tests - CodeAuditor
-  condition: succeeded()
-  jobs:
-  - job: Test
-    steps:
-    - template: templates/test.yml
\ No newline at end of file
+    - template: templates/build.yml
```

Comment:
Let's do this after we build out the other pipeline

Reply:
Done